tools/compstrat/src/preprocessing.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+
+STRAT1_PREFIX = "strat1_"
+STRAT2_PREFIX = "strat2_"
+
+
+def preprocess(
+    strat1_runs: list[pd.DataFrame], strat2_runs: list[pd.DataFrame]
+) -> tuple[pd.DataFrame, pd.DataFrame]:
+    all_runs = pd.concat(
+        strat1_runs + strat2_runs,
+        axis=1,
+        join="inner",
+        keys=list(map(lambda num: f"{STRAT1_PREFIX}{num}", range(len(strat1_runs))))
+        + list(map(lambda num: f"{STRAT2_PREFIX}{num}", range(len(strat2_runs)))),
```

Comment:
```suggestion
        + [f"{STRAT2_PREFIX}{num}" for num in range(len(strat2_runs))]
```

turns out, this is more pythonic, shorter and arguably easier to read